shared/domain/src/commonMain/kotlin/network/bisq/mobile/domain/replicated/security/pow/ProofOfWorkVO.kt

```diff
@@ -0,0 +1,16 @@
+package network.bisq.mobile.domain.replicated.security.pow
+
+import kotlinx.serialization.Serializable
+import network.bisq.mobile.domain.utils.hexToByteArray
+
+@Serializable
+data class ProofOfWorkVO(
```

Comment:
nit - VO here stands for Value Object right? Wouldn't DTO be more appropriate?

Reply:
I had earlier Dto, but as the models are used as domain models VO is more appropriate as DTO is more for data transfer. On the Bisq 2 side they are called DTO as there their only purpose is to serve for data transfer.